features/fetch-crl/config.pan

```diff
@@ -3,6 +3,10 @@ unique template features/fetch-crl/config;
 
 variable FETCH_CRL_QUIET ?= true;
 variable FETCH_CRL_FORCE_OVERWRITE ?= true;
+# This avoid having fetch-crl-boot returning a non-zero exit status on
```

Comment:
This avoid**s** having fetch-crl-boot **return**<strike>ing</strike> a non-zero exit status on